### DISCUSSION TITLE
On receiving device (PC) ask for download folder

### DISCUSSION THREAD

**Discussion by mopperle** [localsend/localsend#1817] [Discussion (English, International)] [answered]
I mainly use LocalSend for transfering photos and videos to my Win11 PC. They are stored in different folders for further processing. So when sending photos/videos (or other files) it would be nice to be asked where to store the current transfer instead of the default folder.

**Comment by alicemorgan33** [comment]
That’s a great suggestion! Having the option to choose the destination folder for transfers would definitely streamline your workflow and make file organization much easier. It could enhance the overall user experience with LocalSend!

**Comment by Tienisto** [comment, marked as answer]
If you tap on "Options" when the receive page is shown, then you can change the download folder ad-hoc.

**Reply by mopperle** [reply]
Perfect! Didnt know that. BTW, is there a kind of help(file)?

**Reply by Tienisto** [reply]
@mopperle no not yet

**Reply by mopperle** [reply]
No probem, you deserve a donation anyway. ;)

**Reply by TheCDB** [reply]
Unfortunately not on the google chromecast with google tv. Changing the default download folder is not possible. (No app available for this) ?
